lib/ansible/modules/clustering/k8s/k8s_exec.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# Copyright 2019 Red Hat | Ansible
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: k8s_exec
+short_description: Execute command in Pod
+version_added: "2.10"
+author: "Tristan de Cacqueray (@tristanC)"
+description:
+  - Use the Kubernetes Python client to execute command on K8s pods.
+extends_documentation_fragment:
+  - k8s_auth_options
+requirements:
+  - "python >= 2.7"
+  - "openshift == 0.4.3"
+  - "PyYAML >= 3.11"
+options:
+  proxy:
+    description:
+    - The URL of an HTTP proxy to use for the connection. Can also be specified via K8S_AUTH_PROXY environment variable.
+    - Please note that this module does not pick up typical proxy settings from the environment (e.g. HTTP_PROXY).
+    type: str
+  namespace:
+    description:
+    - The pod namespace name
+    type: str
+    required: yes
+  pod:
+    description:
+    - The pod name
+    type: str
+    required: yes
+  container:
+    description:
+    - The container name
```

Comment:
Something like  `- The name of the container in the pod to connect to. Defaults to only container if there is only one container in the pod.` might be a littel clearer, and matches the upstream kubernetes client docs.

Reply:
done, thanks for the suggestion.